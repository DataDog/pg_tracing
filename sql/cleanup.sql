-- Drop test table and test functions
DROP TABLE pg_tracing_test;
DROP function test_function_project_set;
DROP function test_function_result;
DROP EXTENSION pg_tracing CASCADE;
