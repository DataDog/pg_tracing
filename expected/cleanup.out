-- Drop test table and test functions
DROP TABLE pg_tracing_test;
DROP function test_function_project_set;
DROP function test_function_result;
DROP EXTENSION pg_tracing CASCADE;
NOTICE:  drop cascades to 2 other objects
DETAIL:  drop cascades to view peek_spans_with_level
drop cascades to view peek_ordered_spans
